common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -26,6 +26,26 @@ trait ABTestSwitches {
     exposeClientSide = true
   )
 
+  Switch(
+    ABTests,
+    "ab-contributions-epic-articles-viewed",
+    "States how many articles a user has viewed in the epic",
+    owners = Seq(Owner.withGithub("jranks123")),
+    safeState = Off,
+    sellByDate = new LocalDate(2020, 1, 27),
+    exposeClientSide = true
+  )
+
+  Switch(
+    ABTests,
+    "ab-contributions-epic-country-name",
+    "Displays country name in the epic",
+    owners = Seq(Owner.withGithub("jranks123")),
```

Comment:
Small detail but should you be the owner @tomrf1 or maybe Iona?

Reply:
good question, I'm not even sure what this is for

Reply:
My understanding is it's the first person you should go to if there's a problem with the test, to stop the test, or if you want information about the test

Reply:
I'm more than happy to be the first person you should go to if you have questions about the test, as in context. But perhaps not best technically. Could both Tom and I be on there do you reckon?